profile/index.php

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>VITA lincoln</title>
+	<?php
+		$root = realpath($_SERVER["DOCUMENT_ROOT"]);
+		require_once "$root/server/header.php";
+	?>
+	<link href="/assets/css/main.css" rel="stylesheet">
+	<link href="/assets/css/form.css" rel="stylesheet">
+</head>
+<body>
+	<?php
+		$page_subtitle = 'Volunteer Profile';
+		require_once "$root/components/nav.php";
+	?>
+  <div>
+    <div class="container">
+      <div class="row justify-content-center">
+        <div class="col col-12 col-sm-8">
+              <h2 class="form-title" id="nameHeaderProfile"></h2>
+              <h4>Name</h4>
+              <p id="nameProfile"></p>
+              <h4>Phone Number</h4>
+              <p id="phoneProfile"></p>
+              <h4>Email</h4>
+              <a class="emailProfileStatic" href="mailto:joe@example.com?subject=feedback" id="emailProfile"></a>
```

Comment:
Hm so I don't think having this mailto is necessary. So this profile page... I hadn't planned on letting any volunteer see another volunteers. This was mostly a private thing for a single volunteer to update their information. It looks like you're pursuing a route more so of anyone can see anyone? If you're doing the latter, it's a little bit more involved than what you're doing right now; so I would suggest going with the former. And if you go with the former, we probably don't need the mailto on this, but I have no problem leaving it, I just wanted to clarify the uses of this page.